yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
+    min_zoom: CASE WHEN to_number(tags->'height', '9999.99') < 10 THEN 15
```

Comment:
Hmm, `height` is only in `tags` on my local machine?

Agree simple `to_number` SQL isn't the best to capture variable null, meter, and feet `height` units.  We'll have the similar problem for peak heights?

Should there be a post processing step that converts the waterfall height to implied meters, and does a cull? The downside is we'd have to maintain the height logic cutoffs in two places, so only select all waterfalls starting at zoom 12 in the YAML here, and move the zoom logic to the new transform, after the value is cast into meters?